src/main/java/core/basesyntax/service/TransactionParser.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Transaction;
+
+public class TransactionParser {
+    public Transaction parseLine(String line) {
+        String[] parts = line.split(",");
```

Comment:
It's good practice to define the comma separator as a constant to avoid magic strings. This way, if the separator changes, you only need to update it in one place.